src/io/unix/iouring.rs

```diff
@@ -197,3 +197,37 @@ impl File {
         Ok(Self { sync, rio })
     }
 }
+
+
+/// Trait implementation to track that all required function are implemented
+#[async_trait::async_trait]
+impl super::super::FileTrait for File {
```

Comment:
Why not move implementations into trait?

Reply:
I've added an additional comment on top of each implementation explaining the reason why it should not contain actual function body inside

Reply:
https://blog.rust-lang.org/inside-rust/2023/05/03/stabilizing-async-fn-in-trait.html